core/db/migrate/20160511072249_change_collation_for_tag_names.rb

```diff
@@ -0,0 +1,9 @@
+# This migration is added to circumvent issue #623 and have special characters
+# work properly
+class ChangeCollationForTagNames < ActiveRecord::Migration
+  def up
+    if ActsAsTaggableOn::Utils.using_mysql?
+      execute("ALTER TABLE spree_tags MODIFY name varchar(255) CHARACTER SET utf8 COLLATE utf8_bin;")
```

Comment:
@krzysiek1507 I'm not getting why it is executing migrations from gem and if it is then it should have run all migration from gem